ui/src/api/starter.ts

```diff
@@ -42,10 +42,11 @@ export type StarterRequest = {
   builder: Builder;
 };
 
+export const serverAddress = !process.env.REACT_APP_SERVER_ADDRESS
+  ? 'https://adopt-tapir.softwaremill.com'
+  : process.env.REACT_APP_SERVER_ADDRESS;
```

Comment:
You can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here, assuming you're using Node 14.0.0 or higher
```
export const serverAddress = process.env.REACT_APP_SERVER_ADDRESS ?? 'https://adopt-tapir.softwaremill.com'
```

Reply:
done